tao_compiler/mlir/disc/BUILD

```diff
@@ -420,6 +420,27 @@ cc_library(
     alwayslink = 1,
 )
 
+cc_library(
+    name = "rng_rewriter",
```

Comment:
maybe we have other rewriter patterns from mhlo to disc-mhlo , so maybe we can rename to `disc_mhlo_rewirter`?

Reply:
done